examples/fit/pipelines/infer_pipeline.py

```diff
@@ -0,0 +1,178 @@
+from typing import Tuple
+
+import numpy as np
+from diffusion import create_diffusion
+
+import mindspore as ms
+from mindspore import Tensor, ops
+
+from mindone.models.modules import get_2d_sincos_pos_embed, precompute_freqs_cis_2d
+
+
+class FiTInferPipeline:
+    """
+
+    Args:
+        dit (nn.Cell): A `FiT` to denoise the encoded image latents.
```

Comment:
correct `dit` to `fit`, or more general name `model`

Reply:
Fixed. Thx